### PR TITLE
Support XCLogParser

### LINKED ISSUE #24: Support XCLogParser output
https://github.com/spotify/XCLogParser can produce a json output with error lines. Worth investigating how to support it.

### PR DESCRIPTION
**Pull request by edigaryev**
Test fixture uses JSON output from https://github.com/edigaryev/cirrus-ci-annotation-examples/pull/2.

Resolves #24.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,9 @@ This module helps Cirrus CI to parse artifacts and find relative annotations to
 * Ruby
   * RSpec
   * RuboCop
+* Swift
+  * XCLogParser
+* JetBrains Qodana
```
Comment (fkorotkov) [inline README.md:19]: maybe add a link https://github.com/JetBrains/Qodana?

```suggestion
* [JetBrains Qodana](https://github.com/JetBrains/Qodana?)
```

File: README.md [hunk 2]
```diff
@@ -14,6 +14,9 @@ This module helps Cirrus CI to parse artifacts and find relative annotations to
 * Ruby
   * RSpec
   * RuboCop
+* Swift
+  * XCLogParser
```
Comment (fkorotkov) [inline README.md:18]: Maybe also a link to https://github.com/spotify/XCLogParser

```suggestion
  * [XCLogParser](https://github.com/spotify/XCLogParser)
```

Comment (edigaryev) [reply README.md]: See https://github.com/cirruslabs/cirrus-ci-annotations/pull/27/commits/14d3e8c1319e2567f81880a349bd9a4306583b87.

File: README.md (hunk 1 above)
Comment (edigaryev) [reply README.md]: See https://github.com/cirruslabs/cirrus-ci-annotations/pull/27/commits/14d3e8c1319e2567f81880a349bd9a4306583b87.

**Review by fkorotkov — APPROVED** [review]

**Pull request merged**